### ISSUE TITLE
Obnoxious noise when opening inventory

### ISSUE THREAD

**Issue by jackm7195** [RubixDev/Inventorio#350]
I'll keep this basic, inventorio plays the sound of a bow shooting an arrow when you open the inventory, and it can't really be controlled so it is just way too loud for my liking. Any other sound management mods I've used show that inventorio doesn't even have a sound associated with it? If there is something I'm overlooking to change/ remove the sound? Please, let me know, but its just a very annoying hiccup for an otherwise amazing mod I can't play without\

I play on fabric 1.20.4, if that helps

**Comment by VaDeRvR** [comment]
Little delayed response but i figured out that the issue was caused by the Sounds mod (https://www.curseforge.com/minecraft/mc-mods/sound). It’s clearly an incompatibility issue because even when you disable every sound that this mod adds, the problem still occurs. It's unfortunate because I really love both of these mods

**Comment by jackm7195** [comment]
Yeah I did some trial and error and figured that out, but I do really
appreciate you taking the time to let me know. Such a shame, since like you
said it’s hard to not love both mods. Again though, thank you

**Comment by X66Herobrine66X** [comment]
+1, really hope this gets fixed!

**Comment by Carloyaboyy** [comment]
> +1, really hope this gets fixed!

I managed to fix this by going into the sounds mod config (ui.json) and setting its "inventoryOpenSoundEffect"'s Shouldplay value to false. 
Its a little missing sound, but the rest of the sounds mod still works that way. 
Hope this helps

**Comment by YnwLNE** [comment]
> > +1, really hope this gets fixed!
> 
> I managed to fix this by going into the sounds mod config (ui.json) and setting its "inventoryOpenSoundEffect"'s Shouldplay value to false. Its a little missing sound, but the rest of the sounds mod still works that way. Hope this helps

is this with the fabric version of both mods? I tried doing this and it didn't work

**Comment by ac0mplex** [comment]
This issue has been posted before to Sounds mod: https://github.com/IMB11-Mods/Sounds/issues/142. However it got closed because 1.20.1 is no loger supported so it won't get fixed for this version unfortunately.

I've reposted the issue for 1.21.1: https://github.com/IMB11-Mods/Sounds/issues/230

**Comment by IgnaNobody** [comment]
> > > +1, really hope this gets fixed!
> > 
> > 
> > I managed to fix this by going into the sounds mod config (ui.json) and setting its "inventoryOpenSoundEffect"'s Shouldplay value to false. Its a little missing sound, but the rest of the sounds mod still works that way. Hope this helps
> 
> is this with the fabric version of both mods? I tried doing this and it didn't work. This is really driving me insane for how annoying and little the bug is. I'm on forge 1.20.1

Hi!, Were you able to fix it?
